src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/NomisData.kt

```diff
@@ -412,6 +418,27 @@ class NomisData(
           builder.apply(dsl)
         }
     }
+
```

Comment:
added this under a global since it requires multiple cases and yet isn't directly linked to a booking or offender